uspace/lib/c/generic/private/futex.h

```diff
@@ -146,7 +169,7 @@ static inline errno_t futex_down_composable(futex_t *futex,
 static inline errno_t futex_up(futex_t *futex)
 {
 	if (atomic_fetch_add_explicit(&futex->val, 1, memory_order_release) < 0)
-		return __SYSCALL1(SYS_FUTEX_WAKEUP, (sysarg_t) futex);
```

Comment:
Actually, this is not correct, the wakeup might need to allocate the handle here too. But what to do with a failure to allocate it? Going to switch to the allocate-before-decrement in sleep strategy.